source/modules/sensors/SensorsGetters.mc

```diff
@@ -422,5 +423,42 @@ module SensorsGetters {
         function getActiveMinutesWeekGoal() as Number? {
             return ActivityMonitor.getInfo().activeMinutesWeekGoal;
         }
+
+        function getSunRiseSet() as Array<Object?>? {
```

Comment:
Could you please add a description to this method (or a couple of comments), it's just a bit hard to get the idea how it works at each step